### DISCUSSION TITLE
What is "Assign Email" supposed to do/doing

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16047] [How to] [answered]
What is the command "Database / Email History / List" -> Actions -> Assign Email supposed to do?
When select one or more e-mails and select the command, and empty green status box opens to the right.
I was hoping this function would allow me to mass assign e-mails to leads, opportunities, projects, or service contracts.

**Comment by rskrzypczak** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10221999/168725306-e7f1b254-6b40-4248-9bb0-eec7f5786368.png)
The action causes the mail scanner action to be performed again for the selected e-mails.
